### ISSUE TITLE
Feature Request: relay mode changes, quit reasons and join addresses

### ISSUE THREAD

**Issue by acerspyro** [reactiflux/discord-irc#345]
Hi, we are using Discord-IRC in a support channel for our project.

A lot of our users have noticed that the bot does not relay kicks, bans, mode changes or topic changes. It's useful to know when people are gone so you don't end up talking to yourself :P.

**Comment by ekmartin** [comment]
Kicks and bans seem like a reasonable addition to have behind the `ircStatusNotices` flag. There's an issue for topic changes here: https://github.com/reactiflux/discord-irc/issues/220

**Comment by rnyberg** [comment]
Yes to topics, please no to quits/joins (at least not visible by default).

**Comment by ekmartin** [comment]
@rnyberg: quits/joins are already supported behind the `ircStatusNotices` flag (off by default).

**Comment by rnyberg** [comment]
I meant that while having `ircStatusNotices`show kicks and bans is great, I wouldn't want the same setting to enable quits/joins.

**Comment by ekmartin** [comment]
Hm, I see. Any particular reason why? I would think that people that are interested in seeing quits/joins would be interested in seeing kicks and bans as well, but I might be wrong.

**Comment by rnyberg** [comment]
It's really spammy. A lot of IRC users hide joins/parts/quits because it's mostly redundant noise, especially on bigger channels. Add a few users with dodgy connections constantly ping timeout reconnect looping and it's just horrid.

Topics, kicks and bans however aren't commonplace, and rarely just noise, either.
